jsapp/js/account/plan.component.tsx

```diff
@@ -27,7 +27,8 @@ import Button from 'js/components/common/button';
 import classnames from 'classnames';
 import LoadingSpinner from 'js/components/common/loadingSpinner';
 import {notify} from 'js/utils';
-import {BaseProduct} from "js/account/subscriptionStore";
+import {BaseProduct} from 'js/account/subscriptionStore';
```

Comment:
I know you're just running prettier, but this import isn't being used. We should remove it.